### ISSUE TITLE
pyse.py compatibility w AARTFAAC

### ISSUE THREAD

**Issue by gijzelaerr** [transientskp/tkp#389]
The following issues need to be addressed in order to get pyse.py to work with AARTFAAC images:

 * remove pixel size dependance for AA images
 *  for wcs, AARTFAAC info is in [alt/az]- convert this to RA/Dec
 *  beam- is the beam needed, or can we use peak flux? would relying on peak flux result in stretched sources? or could we find islands just knowing the peak flux?
 *  continuing discussion about the beam, what about extracting non-detections?

**Comment by hsuyeep** [comment]
Comments on a couple of issues from above. The rest need more discussion.
* Current WCS implementation already provides RA/DEC values for each pixel, instead of alt/az.
* The beam can be just based on the UV sampling. Islands can be found just knowing the peak flux.. Sources away from the zenith will not be stretched due to the SIN projection used.

**Comment by hsuyeep** [comment]
- I guess pixel size dependence was related to the pixels further away from the zenith representing a larger region of the sky. This will still be the case due to the projection. The WCS takes care of converting each pixel to the correct RA/DEC. Away from the zenith, we end up with a sparsely sampled RA/DEC region, but with each pixel containing power from the sky surrounding the pixel as well. We can interpolate to get continuity there (needed only in RA/DEC space).
- alt/az to RA/DEC: Already in place.
- Whether PSF is needed, or can peak flux be used: I guess this refers to source flux extraction. It should not be a problem for point sources at low elevations, since such sources will still occupy the same number of pixels as when they are close to zenith (since the PSF maintains its shape in pixels in the projected image).
- Nondetections: Away from the zenith, a non detection from a group of pixels at the location of the source would have a higher pixel value, as the noise here is higher. This is expected behaviour. The nature of the noise does not change, so the same technique can be applied over the full elevation range.

**Comment by gijzelaerr** [comment]
so I'm having a bit of a hard time extracting the TODO's here. Can you rephrase with what we actually should do or what still requires discussion and with who?

**Comment by hsuyeep** [comment]
A couple of TODOs remain, to be resolved via  further investigation, and not really via pyse code changes. pyse.py is otherwise already working fine with AARTFAAC images. I have copied your original queries below:
- remove pixel size dependance for AA images: **Not required for pyse working, to be accounted for during analysis.**
- for wcs, AARTFAAC info is in [alt/az]- convert this to RA/Dec: **Already in place and working with pyse.**
- beam- is the beam needed, or can we use peak flux? would relying on peak flux result in stretched sources? or could we find islands just knowing the peak flux?" **To be investigated.** 
Whether the PSF should be forced during source extraction, or whether the peak flux can be used would be dependent on the SNR of the source, and its shape distortion due to ionospheric effects. The discriminator should be source flux extraction performance, probably of calibrator sources or via simulations (to check this with Dario).
- continuing discussion about the beam, what about extracting non-detections? **Nothing to do.** pyse will extract non-detections in the same way everywhere in the image. A related issue is whether it should return the forced fit noise estimate, or the pixel value. I feel since the integrated flux (in Jy) of an unresolved source should be the same as its peak flux (in Jy/beam), the non-detection can return the pixel value at the expected location of the source as an upper limit of the source flux.

**Comment by gijzelaerr** [comment]
ok so in short we need to discuss the 'is the beam needed, or can we use peak flux' issue this Friday and then act according to what comes from that.

**Comment by gijzelaerr** [comment]
all issues has been discussed and there is no issue!
